readme.md

```diff
@@ -138,6 +139,34 @@ Note that you need to tell the client the server's port
 (unless you're using the default server port, which is 3030)
 so the client knows where he can find the server.
 
+### Settings
+
+#### authenticated
+
+This setting is used to specify which link(s) should be visible to all users
+or only to authenticated users or to unauthenticated users.
+Let's consider the `Footer` configuration below;
```

Comment:
I'd change this description to be generic, since no configuration setting was explained, we should try to explain to users what these settings are about: specify links in 3 different components and specify if they should be shown to any user, only authenticated users or only unauthenticated users.